assets/translations/en-us.yaml

```diff
@@ -4568,6 +4586,476 @@ workload:
+      upgradeNode: Upgrading Nodes
+      upgradeSysService: Upgrading System Services
+    version: Version
+  host:
+    label: Hosts
+    promote:
+      none: ' '
+      running: promoting
+      failed: promote failed
+      unknown: promote halted
+      promoteRestart: restarting
+      promoteSucceed: promote completed
+    tabs:
+      network: Network
+      overview: Overview
+      basics: Basics
+      instance: Virtual Machines
+      monitor: Monitor Data
+      disk: Disks
+    detail:
+      title:
+        network: Network Configuration
+      hostIP: Host IP
+      role: Role
+      os: Operating System
+      create: Creation Time
+      update: Last Update Time
+      customName: Custom Name
+      consoleUrl: Console URL
+      more: More Information
+      uuid: UUID
+      containerRuntime: Container Runtime
+      kernel: Kernel
+      management: Management Node
+      compute: Compute Node
+      storage: Storage Size
+      nic: Physical NIC
+      notRecommended: 'Not recommended'
+      networkType: Type
+    enableMaintenance:
+      title: Enable Maintenance Mode
+      protip: The operation will migrate all virtual machines on this node to other nodes.
+    cordon:
+      title: Cordon
+      protip: This operation will mark the node { node } as unschedulable.
+    disk:
+      add: Add Disk
+      path:
+        label: Path
+      storageReserved:
+        label: Storage Reserved
+      allowScheduling:
+        label: Scheduling
+      evictionRequested:
+        label: Eviction Requested
+
+  virtualMachine:
+    label: Virtual Machines
+    instance:
+      singleInstance: 
+      multipleInstance: 
+      single:
+        label: Single Instance
+        nameLabel: Name
+        host: 
+          label: Host Name
+          placeholder: default to the virtual machine name.
+      multiple:
+        label: Multiple Instance
+        nameNsDescription: Name prefix for each instance
+        count: Count
+        countTip: Count should be between 1 and 10
+        nameLabel: Name Prefix
+        host: 
+          label: Host Prefix Name
+          placeholder: default to the virtual machine name.
+    useTemplate:
+      label: "Use VM Template:"
+      template:
+        label: template
+      version:
+        label: version
+    console:
+      novnc: Open in Web VNC
+      serial: Open in Serial Console
+    promptRemove:
+      title: 'Select the volume you want to delete:'
+      deleteAll: Delete All
+    restartTip: Restart the virtual machine now to take effect of the configuration changes.
+    createRunning: Start virtual machine on creation
+    restartNow: Restart Now
+    createSSHKey: Create SSH Key
+    installAgent: Install guest agent
+    enableUsb: Enable USB Tablet
+    sshTitle: Add Public SSH Key
+    imageTip: An external URL to the .iso, .img, .qcow2 or .raw that the virtual machine should be created from.
+    volume:
+      dragTip: Try drag and drop to change bootOrder.
```

Comment:
Phrasing `Drag and drop volumes, or use the volume's arrows, to change the boot order`

Reply:
Done